extensions/GlobalMessages/GlobalMessagesA.i18n.php

```diff
@@ -34,12 +34,12 @@
 * Expiry of block: $6
 * Intended blockee: $7
 
-You may contact $1 or one of the other [[{{MediaWiki:Grouppage-sysop}}|administrators]] to discuss the block.
-
 Note that you may not use the "e-mail this user" feature unless you have a valid e-mail address registered in your [[Special:Preferences|user preferences]] and you have not been blocked from using it.
 
 Your current IP address is $3, and the block ID is #$5.
-Please include all above details in any queries you make.',
+
+You may contact [[Special:Contact|Fandom Support]] or an [[Special:ListUsers|{{MediaWiki:group-sysop-member}}]] to discuss the block.
```

Comment:
I believe this will also affect local block notices ("You may contact $blockingAdminName or one of the other administrators..."). Is this intended?